src/components/Modal/ProfileModal.tsx

```diff
@@ -111,6 +141,33 @@ export class ProfileModal extends React.Component<{
               gap: '10px',
             }}
           >
+            {this.props.discordUsername && this.props.discordDiscriminator ? (
+              <Button
+                icon
+                labelPosition="left"
+                fluid
+                color="red"
+                animated="fade"
+                onClick={this.deleteDiscord}
+              >
+                <Icon name="discord" />
+                <Button.Content visible>Unlink Discord Account</Button.Content>
+                <Button.Content
+                  hidden
+                >{`${this.props.discordUsername}#${this.props.discordDiscriminator}`}</Button.Content>
+              </Button>
+            ) : (
+              <Button
+                icon
+                labelPosition="left"
+                fluid
+                color="orange"
+                onClick={this.authDiscord}
+              >
+                <Icon name="discord" />
+                Link Discord Account
```

Comment:
let's say "Link Discord Account/Get Subscriber Role"?

Reply:
That looked a little awkward because it's too long. I've added a popup instead.